src/AppInstallerCLIPackage/Package.appxmanifest

```diff
@@ -16,7 +16,6 @@
   <Dependencies>
     <!-- Minimum supported version is 1809 (October 2018 Update, aka RS5) -->
     <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.17763.0" MaxVersionTested="10.0.19033.0" />
-    <PackageDependency Name="Microsoft.VCLibs.140.00.UWPDesktop" MinVersion="14.0.25426.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />

```

Comment:
><PackageDependency Name="Microsoft.VCLibs.140.00.UWPDesktop" MinVersion="14.0.25426.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" /> [](http://example.com/codeflow?start=4&length=182)

Is this removal intentional?

Reply:
Yes, it is being injected by VS anyway.  This one was leading to debug needing the release package (as VS left this and injected the debug on as well).